### DISCUSSION TITLE
官网下载的X86 no-avx2安装包仍然无法安装

### DISCUSSION THREAD

**Discussion by dzygoon** [apache/doris#22595] [RFC]
因为服务器的CPU不支持avx2指令集，所以想安装no-avx2的安装包，但是从官网选择下载x86 no-avx的安装包进行安装时，启动be仍然报错The CPU does not support AVX2 instruction set，cpu不支持avx2，请问如何解决这个问题？

**Comment by LemonLiTree** [comment]
which version?
